decksite/league.py

```diff
@@ -132,7 +133,7 @@ def active_league():
     return guarantee.exactly_one(leagues)
 
 def insert_match(params):
-    match_id = db().insert("INSERT INTO `match` (`date`) VALUES (DATE_FORMAT(now(), '%%s'))")
```

Comment:
Could also use UNIX_TIMESTAMP() to keep it (my)SQL. Does this work the same regardless of timezone? I assume so but if not we could also use our home grown dtutils.dt2ts(dtutils.now()).